### PR TITLE
Update readme contents and rename license file

### PR DESCRIPTION
**Pull request by arnellebalane**
## Issue

#15 Improve readme contents

## Changes

1. Link to license file from the readme
1. Re-organize the code example in the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,61 +2,60 @@
...
 repositories {
-    mavenCentral()
+  mavenCentral()
 }
 
-dependencies {
-    implementation "org.idpass:idpass-lite-java:0.0.1-SNAPSHOT"
-    implementation 'com.google.protobuf:protobuf-java:3.12.2'
+implementation "org.idpass:idpass-lite-java:0.0.1-SNAPSHOT"
+implementation "com.google.protobuf:protobuf-java:3.12.2"
```
Comment (jeremi) [inline README.md]: Why removing the indentation?

**Review by jeremi — CHANGES_REQUESTED** [review]
Did someone test the sample code?
Would be cool if there was a way to have this code as part of the test suite ;)

File: README.md (hunk 1 above)
Comment (arnellebalane) [reply README.md]: Oops this was a mistake, already fixed in latest commit. Will coordinate with @typelogic to add the sample code in the README into the test suite as well.

**Review by typelogic — CHANGES_REQUESTED** [review]
I just merge latest changes to develop branch which contains an `android/` folder to build an equivalent `aar` library for Android. Hence, this repo now builds two types of libraries:
-  A `jar` library for general Java usage
- An `aar` library for Android usage

Current PR is ok. Just requesting to add this new additional Android `aar` build.

**Review by typelogic — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (arnellebalane) [reply README.md]: Sample code now added to the test suite, thanks @typelogic ! 👍

**Review by jeremi — APPROVED** [review]

**Pull request merged**